### ISSUE TITLE
Example server and client not working

### ISSUE THREAD

**Issue by p13rr0m** [modelcontextprotocol/python-sdk#78]
**Describe the bug**
The example servers, in the folder `examples/servers`, in combination with the clients from README.md do not work. 

1. Using mcp_simple_tool as an example, starting the servers with the command `mcp-simple-tool` or `mcp-simple-tool --transport sse --port 8000` is not correct. I believe that should be `uv run mcp-simple-tool` or  `uv run mcp-simple-tool --transport sse --port 8000` respectively.

2. The client code is not able to talk to the server. I am not sure which method (stdio vs sse) the example should be using. Additionally, the import for the client does not exist.

```python
from mcp.client import ClientSession

# ClientSession
async with ClientSession() as session:
    await session.initialize()

    # List available tools
    tools = await session.list_tools()
    print(tools)

    # Call the fetch tool
    result = await session.call_tool("fetch", {
        "url": "https://example.com"
    })
    print(result) 
```

**Comment by hemangjoshi37a** [comment]
same problem here in the #80

**Comment by hemangjoshi37a** [comment]
let me know please if you get this working

**Comment by tanhaipeng** [comment]
I also encountered many problems while studying and needed to improve the quality of documents.

**Comment by hemangjoshi37a** [comment]
I just found out that the MCP is useless to me as i use Ubuntu Linux and the MCP is only supported through the claude app. very sad .

**Comment by dsp-ant** [comment]
Fixed in 44ec763

**Comment by tanhaipeng** [comment]
> Fixed in [44ec763](https://github.com/modelcontextprotocol/python-sdk/commit/44ec763f929373a242937d145cbeb6e3503f034c)

please add sse protocol examples!!!
